src/services/search/service/search-index.ts

```diff
@@ -35,6 +36,7 @@ export type GSearchIndex = {
   size_in_mb: number;
   num_subjects: number;
   num_entries: number;
+  status: 'open' | 'delete_pending';
```

Comment:
Missed one - `| string`; But based on "It is otherwise the same as an index document (as fetched from [Index Show](https://docs.globus.org/api/search/reference/index_show))."

https://docs.globus.org/api/search/reference/index_list/#indexwithpermissions

I wonder if `IndexWithPermissions` should just be:

```ts
IndexWithPermissions = GSearchIndex & {
  permissions: string[];
}
```

Reply:
Yeah probably should be, that's how I defined it in my temp type in the webapp